### PR TITLE
modified the calibration scripts

### PR DESCRIPTION
**Pull request by wfgsy101**
modified the calibration scripts

### CODE REVIEW

File: modules/tools/calibration/process.py [hunk 1]
```diff
@@ -48,12 +48,12 @@ def get_start_index(data):
...
 
-    start_ind = np.where(data['brake_percentage'] == 40)[0][0]
+    start_ind = np.where(data['vehicle_speed'] == 0.0)[0][0]
 
     ind = start_ind
     while ind < len(data):
-        if data['brake_percentage'][
-                ind] == 40:  #or data['vehicle_speed'][ind] == 0.0:
+        if data['vehicle_speed'][
+                ind] == 0.0:  #or data['vehicle_speed'][ind] == 0.0:
```
Comment (xiaoxq) [inline modules/tools/calibration/process.py]: Put comment in another line to make the code easy to read.

File: modules/tools/calibration/process_data.sh [hunk 2]
```diff
@@ -20,7 +20,7 @@
 
 DIR="$( cd "$( dirname "${BASH_SOURCE[0]}"  )" && pwd  )"
 rm $DIR/result.csv
-for f in `ls ${1}/*_recorded.csv`
+for f in `ls ${1}/*r0.csv`
```
Comment (xiaoxq) [inline modules/tools/calibration/process_data.sh]: Avoid single-char variable name.

File: modules/tools/calibration/process.py [hunk 3]
```diff
@@ -48,12 +48,12 @@ def get_start_index(data):
     if np.all(data['vehicle_speed'] == 0):
         return 0
 
-    start_ind = np.where(data['brake_percentage'] == 40)[0][0]
+    start_ind = np.where(data['vehicle_speed'] == 0.0)[0][0]
```
Comment (xiaoxq) [inline modules/tools/calibration/process.py]: Not safe to compare `equal` on float numbers.

Comment (wfgsy101) [reply modules/tools/calibration/process.py]: modified as your require

File: modules/tools/calibration/process_data.sh (hunk 2 above)
Comment (wfgsy101) [reply modules/tools/calibration/process_data.sh]: has been changed to t*r0.csv

**Pull request merged**